python_modules/dagster/dagster/_generate/download.py

```diff
@@ -6,10 +6,10 @@
 import click
 import requests
 
+from dagster.version import __version__ as dagster_version
```

Comment:
can we instead get version from the current python env?

Reply:
can you elaborate? by my current interpretation that is already true - we are importing `dagster` in the current python environment and getting what version it is

Reply:
nvm - i misunderstood `dagster.version`.